.pre-commit-config.yaml

```diff
@@ -38,4 +38,5 @@ exclude: |
     ^lib/jsmn/|
     ^lib/Catch2/|
     ^lib/libsysbase/
+    ^lib/Middlewares/Third_Party/littlefs/|
```

Comment:
This regex does not look right 🧐 (see the line endings…)

Reply:
Thanks :)